server/src/main/java/com/cloud/network/guru/PublicNetworkGuru.java

```diff
@@ -74,7 +78,15 @@ public class PublicNetworkGuru extends AdapterBase implements NetworkGuru {
     @Inject
     Ipv6Service ipv6Service;
     @Inject
-    NetworkModel networkModel;
+    protected NetworkModel networkModel;
+    @Inject
+    protected VpcDao vpcDao;
+    @Inject
+    protected VlanDetailsDao vlanDetailsDao;
+    @Inject
+    protected VpcOfferingDao vpcOfferingDao;
+    @Inject
+    protected VpcOfferingServiceMapDao vpcOfferingServiceMapDao;
```

Comment:
all these dao injections required?

Reply:
it's used in the inherited classes - Netris / NsxPublicNetworkGuru

Reply:
ok